### PR TITLE
upgrade ruby to 3.1.4

### PR DESCRIPTION
**Pull request by wendelfabianchinsamy**
## Purpose

closes: https://github.com/datacite/levriero/issues/160

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -20,10 +20,10 @@ jobs:
       AWS_REGION: ${{ secrets.AWS_REGION }}
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Ruby 2.6
+      - name: Set up Ruby 3.0
```
Comment (jrhoads) [inline .github/workflows/deploy.yml:23]: 3.1

File: .github/workflows/parallel_ci.yml [hunk 2]
```diff
@@ -26,10 +26,10 @@ jobs:
             - name: Checkout Code
               uses: actions/checkout@v3
         
-            - name: Set up Ruby 2.6
+            - name: Set up Ruby 3.0
```
Comment (jrhoads) [inline .github/workflows/parallel_ci.yml:29]: 3.1

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -19,10 +19,10 @@ jobs:
       AWS_REGION: ${{ secrets.AWS_REGION }}
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Ruby 2.6
+      - name: Set up Ruby 3.0
```
Comment (jrhoads) [inline .github/workflows/release.yml:22]: 3.1

File: Gemfile.lock [hunk 4]
```diff
@@ -235,10 +235,9 @@ GEM
     kaminari-core (1.2.2)
     latex-decode (0.4.0)
     link_header (0.0.8)
-    listen (3.1.5)
+    listen (3.0.8)
```
Comment (jrhoads) [inline Gemfile.lock:238]: seems weird that it rolled this back to an earlier verison

**Review by jrhoads — APPROVED** [review]
Small nits in the names of the workflow steps but overall looks good.
Happy to see it move over to ruby 3

**Comment by wendelfabianchinsamy** [comment]
@jrhoads Thank you for the feedback. I've made the changes.

**Review by jrhoads — APPROVED** [review]

**Pull request merged**